### PR TITLE
[BD] Use the latest version of data api in tests

### PR DESCRIPTION
**Pull request by morenol**
## Description https://openedx.atlassian.net/browse/BOM-1359

The analytics_api image used for insights tests is pretty old https://github.com/edx/edx-analytics-data-api/tree/0.28.0

## Reviewers
 - [ ] @andrey-canon
 - [x] Is this ready for edX's review?
- [ ] @jmbowman

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @morenol! I've created [OSPR-4222](https://openedx.atlassian.net/browse/OSPR-4222) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

**Comment by morenol** [comment]
@jmbowman testing locally I noticed that the edx-analytics-data-api version used in the tests is pretty old. I think that we need to update that. What do you think? We can use latest,  ironwood.master, or any other tag in [docker hub](https://hub.docker.com/u/edxops/analytics_api) 

Right now the tests are failing, but I think that to merge https://github.com/edx/edx-analytics-data-api/pull/328 is a good starting point to get these tests working

**Comment by morenol** [comment]
The tests are OK now with the latest docker image of data api

File: .travis.yml [hunk 1]
```diff
@@ -7,22 +7,12 @@ services:
   - docker
 
 env:
-  # Make sure to update this string on every Insights or Data API release
-  DATA_API_VERSION: "0.28.0"
-  DOCKER_COMPOSE_VERSION: "1.9.0"
+  DATA_API_VERSION: "latest"
```
Comment (jmbowman) [inline .travis.yml:10]: We need to remember to make a new release tag and use it here once we're ready to deploy.  Please add that to any task tracking you have for this.

**Review by jmbowman — APPROVED** [review]

**Comment by openedx-webhooks** [comment]
@morenol 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/edx-analytics-dashboard&entry.867055334=https://github.com/edx/edx-analytics-dashboard/pull/972&entry.1484655318=https://github.com/morenol&entry.752974735=2020-03-10+23:01&entry.1917517419=2020-03-12+17:41&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**